addons/mail/static/tests/messaging_menu/messaging_menu.test.js

```diff
@@ -1083,7 +1083,7 @@ test("messaging menu should show new needaction messages from chatter", async ()
         notification_type: "inbox",
         res_partner_id: serverState.partnerId,
     });
-    const [formattedMessage] = await env.services.orm.call("mail.message", "message_format", [
+    const [formattedMessage] = await env.services.orm.call("mail.message", "_message_format", [
```

Comment:
Private method can't (in python) or shouldn't (in mock server) be called by RPC.
This indicates issue (shortcut) in test where instead another method closer to reality should be called.
In this case for example actually sending a message in a flow where it goes to inbox.

Reply:
We can also not do an RPC and call `pyEnv["mail.message"]._message_format(messageId)` directly.

Reply:
It's still not ideal but it's better, it's out of scope of this PR.